### DISCUSSION TITLE
Add "set focus start/end" menu to print statements

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#7132] [Ideas]
We have a wonderful focus start/end context menu:
![image](https://user-images.githubusercontent.com/9154902/172980195-6d12db1d-cdf1-4348-8573-c54efb9d5d46.png)

As @markerikson points out, this would be very useful in our print statements:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/9154902/172979998-c9084936-8185-4906-af60-3e50ff04cd35.png">

Figma:
https://www.figma.com/file/mAotNZRv6M5X0zHIFKPzFY/Replay-Design-Doc?node-id=12807%3A108244

--

Note! This brings up some interesting discussions around entry points. Should this be buried behind a right-click interaction? After all, that's what we do in the console. Or does it deserve its own icon? We have a lot already. Or we could tuck this behind an overflow menu, because that menu is going to come sooner or later :)

cc @souporserious

**Comment by jasonLaster** [comment]
The focus start/end items makes the most sense when you're right clicking on a marker in the timeline because there is an explicit hit you're adding it to.

We could also add it it to the badge as well, but it could be unclear what it is doing. Is it using the Nth hit which is the number or the current paused point? probably current, but that is a bit of a lie.
